src/serac/physics/integrators/wrapper_integrator.cpp

```diff
@@ -26,10 +27,14 @@ void LinearToNonlinearFormIntegrator::AssembleElementGrad(const mfem::FiniteElem
                                                           mfem::ElementTransformation& Tr, const mfem::Vector&,
                                                           mfem::DenseMatrix&           elmat)
 {
-  const mfem::FiniteElement& trial_el = *(trial_fes_.FEColl()->FiniteElementForGeometry(Tr.GetGeometryType()));
-
-  elmat.SetSize(trial_el.GetDof() * trial_el.GetDim(), el.GetDof() * el.GetDim());
-  elmat = 0.;
+  mfem::FiniteElement const* trial_el = trial_fes_.FEColl()->FiniteElementForGeometry(Tr.GetGeometryType());
+
+  if (trial_el) {
+    elmat.SetSize(trial_el->GetDof() * trial_el->GetDim(), el.GetDof() * el.GetDim());
+    elmat = 0.;
+  } else {
+    SLIC_ERROR("Returned trial element is null.");
+  }
```

Comment:
```suggestion
  SLIC_ERROR_IF(!trial_el, "Returned trial element is null.");

  elmat.SetSize(trial_el->GetDof() * trial_el->GetDim(), el.GetDof() * el.GetDim());
  elmat = 0.;
```

Reply:
Sadly, this does not stop the GCC warning.